drivers/DeepSmart/deepsmart/src/lifecycles.lua

```diff
@@ -0,0 +1,123 @@
+local config = require('config')
+local log = require('log')
+local caps = require('st.capabilities')
+local wisers = require('deepsmart.wisers')
+local lifecycle_handler = {}
+
+function lifecycle_handler.init(driver, device)
+  -- report test data
+  --wisers.default_report(driver, device)
+  local is_bridge = wisers.is_device_bridge(device)
+  log.info('device '..device.id..' in init')
+  -- for bridge do not reload devices&&config
+  -- just use the devices&&config loaded from wiser's datastore
+  -- only discover or refresh bridge will reload devices&&config
+  if (is_bridge) then
+    -- just add wiser(bridge) to wisers
+    wisers.add_wiser(device.device_network_id, device:get_field(config.FIELD.IP), device)
+    -- load devices
+    wisers.refresh_wiser(device.device_network_id)
+    -- refresh all devices
+    local devices = wisers.driver:get_devices()
+    for i,v in pairs(devices) do
```

Comment:
Is it possible for there to be multiple bridges in the driver? If so, I think this will not work since the first bridge will be initializing all the devices for the other bridges as well.

Reply:
Check if the device belongs to the current portfolio

Reply:
Ill defer to the testing here